.github/workflows/windows.yml

```diff
@@ -102,6 +102,17 @@ jobs:
             ridk exec patch -p1 -i `
               ..\mroonga\packages\source\patches\mariadb-10.5.5-add-a-missing-export-symbol.diff
           }
+          # mariadb-10.2.36, 10.3.27, 10.4.17, 10.5.8 have a syntax error in CMakeList of mariadb-connector-c.
+          # (mariadb-connector-c is MariaDB client library.)
+          # This bug has already fixed by https://github.com/mariadb-corporation/mariadb-connector-c/commit/242cab8cbcd91af882233730a83627d3b12ced83
+          # However, this fix is not included that above version of mariadb.
+          # Therefore, we fix this bug by using this patch until next release for mariadb.
+          # We will remove this patch when the next version of mariadb will be released.
+          ridk exec curl `
+            -o mariadb-connector-c-fix-syntax-error-in-cmake-3.20.diff `
```

Comment:
`.patch` is better than `.diff` because the file file has more information such as author.

Reply:
I see.
I'll fix it.